templates/index.html

```diff
@@ -332,8 +339,51 @@
+
+                for (var i = 0; i < checkboxes.length; i++) {
+                    if (checkboxes[i].checked) {
+                        var row = checkboxes[i].closest('tr');
+                        var email = row.cells[4].innerText;
+                        if (email) {
+                            emails.push(email);
+                        }
+                    }
+                }
+
+                if (emails.length > 0) {
+                    var emailString = emails.join(',');
+                    var mailtoLink = "mailto:?subject=Acknowledgement of your application to the WEHI student internship program&body=Hi All,%0D%0A%0D%0AThank you for applying for the WEHI Student Internship program.%0D%0A%0D%0AI will be calling you by mobile phone over the next week to get a better understanding of projects that fit your interests and career aspirations. Please let me know if you need to make other arrangements as I can be flexible.%0D%0A%0D%0AYou can see a list of projects as well as other details on the WEHI Student Internship page, where you can click on the project to go to a project page.%0D%0A%0D%0AEach project page has a rough description of the role outline, but the key information is the 'Further Reading' section that may be available for that project.%0D%0A%0D%0APlease also be acquainted with the program handbook.%0D%0A%0D%0ADetails of duties are available via the student projects page at https://wehi-researchcomputing.github.io/students.%0D%0A%0D%0ADetails of the intake dates are available at https://wehi-researchcomputing.github.io/intake_dates.%0D%0A%0D%0AA FAQ is also available at https://wehi-researchcomputing.github.io/faq" + "&to=" + emailString;
+                    window.location.href = mailtoLink;
+                } else {
+                    alert('No students selected.');
+                }
+            });
 
         });
     </script>
+    <div style="display:none" id="email_acknowledgement_text">
+    Hi All,
+
+    Thank you for applying for the WEHI Student Internship program.
+
+    I will be calling you by mobile phone over the next week to get a better understanding of projects that fit your interests and career aspirations. Please let me know if you need to make other arrangements as I can be flexible.
+
+    You can see a list of projects as well as other details on the WEHI Student Internship page, where you can click on the project to go to a project page.
+
+    Each project page has a rough description of the role outline, but the key information is the "Further Reading" section that may be available for that project.
+
+    Please also be acquainted with the program handbook.
+
+    Details of duties are available via the student projects page at https://wehi-researchcomputing.github.io/students.
+
+    Details of the intake dates are available at https://wehi-researchcomputing.github.io/intake_dates.
+
+    A FAQ is also available at https://wehi-researchcomputing.github.io/faq
+
+    </div>
```

Comment:
This might not be necessary anymore. I think it only existed to be used a template for copying to clipboard